core/src/test/scala/akka/persistence/cassandra/snapshot/CassandraSnapshotCleanupSpec.scala

```diff
@@ -18,7 +18,7 @@ import akka.event.Logging
 
 class CassandraSnapshotCleanupSpec extends CassandraSpec {
 
-  private val configPath = "cassandra-plugin"
+  private val configPath = "akka.persistence.cassandra"
```

Comment:
BTW In Alpakka we'd add a `ConfigPath` constant to the `PluginSettings` object.

Reply:
I did some additional cleanup related to that